Add documentation for the accounts app and comments app

lujorryn opened the pull request.

lujorryn committed:
Add documentation for the accounts app and comments app

iamyanbo commented on petpal/comments/views.py:
```diff
@@ -60,7 +73,13 @@ def comment_create_view(request):
             new_comment = Comment(content=content, is_author_seeker=is_author_seeker, seeker=seeker, shelter=shelter, is_review=is_review, rating=rating)
             new_comment.save()
-            return Response({'msg': 'Review Created'}, status=status.HTTP_200_OK)
+            data = {
```
Looks good, I don't think I took into account that a shelter can respond to a review when I made the initial implementation, I think it should work when review is false and no rating but just want to make sure this case is handled

lujorryn replied:
If there's no rating then rating will just be null it won't throw an error.

lujorryn committed:
Fix some more errors in the comments app

1. JsonResponse -> Response
2. Show Shelter doesn't exist if Shelter object not found
3. Modified Response to follow convention
